internal/libpng/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ FetchContent_Declare(libpng
         INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/inst"
         LOG_DIR "${CMAKE_CURRENT_BINARY_DIR}/log"
 )
-
+set(PNG_ARM_NEON "off" CACHE STRING "" FORCE)
```

Comment:
I feel like this is a setting we shouldn't be changing; turning off SIMD for ARM is likely going to result in far poorer performance.

Reply:
We have to do so.

It literally breaks M1.

Unless you're willing to submit a PR to libpng this is our only option for now. That said, it should be in an `if()` to check for M1, and only disable it in that case.

Reply:
With some more research we haven't seem to find a way for apple to automate this without manually adding the flag 
-DCMAKE_APPLE_SILICON_PROCESSOR="arm64"
When creating the build system with cmake .. <flag_here>
We've decided to add an addendum to the README guide on building NovelRT to solve this issue.

Reply:
Adjusted readme and cmake file for libpng to allow people to easily build for apple-silicon